src/gui/advancedsettings.cpp

```diff
@@ -436,6 +439,16 @@ void AdvancedSettings::loadAdvancedSettings()
     addRow(OS_MEMORY_PRIORITY, (tr("Process memory priority (Windows >= 8 only)")
         + ' ' + makeLink("https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/ns-processthreadsapi-memory_priority_information", "(?)"))
         , &m_comboBoxOSMemoryPriority);
+
+    m_spinBoxMemoryWorkingSetLimit.setRange(0, 100);
+    m_spinBoxMemoryWorkingSetLimit.setSingleStep(5);
+    m_spinBoxMemoryWorkingSetLimit.setSuffix(u"%"_qs);
+    m_spinBoxMemoryWorkingSetLimit.setSpecialValueText(tr("Auto"));
+    m_spinBoxMemoryWorkingSetLimit.setValue(static_cast<Application *>(QCoreApplication::instance())->memoryWorkingSetLimit());
+
+    addRow(MEMORY_WORKING_SET_LIMIT, (tr("Process memory working set limit")
```

Comment:
Maybe name it as "Process physical memory(RAM) limit"

Reply:
> Maybe name it as "Process physical memory(RAM) limit"

Then the word "Process" can also be omitted, IMO.